pkg/kubectl/cmd/patch.go

```diff
@@ -175,3 +228,24 @@ func RunPatch(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []stri
 	}
 	return nil
 }
+
+func getPatchedJS(patchType api.PatchType, originalJS, patchJS []byte, obj runtime.Object) ([]byte, error) {
```

Comment:
You're not getting a patched JavaScript so I feel like the method could be two letters longer